powersimdata/data_access/sql/schema.sql

```diff
@@ -20,5 +20,6 @@ CREATE TABLE IF NOT EXISTS scenario_list(
   interval text not null,
   engine text not null,
   runtime text,
-  infeasibilities text
+  infeasibilities text,
+  grid_model text
```

Comment:
Might want this to be `not null` unless we have a default, but it's fine for now

Reply:
Done